### ISSUE TITLE
Desktop canvas size

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#684]
@jmockler, @jhpedemonte, @peller, @childsb, @billreed63

(Couldn't find a bug for this already, which is surprising)

RIght now, desktop canvas size is "auto" (i.e., infinite in both directions). Often, web designers target a particular browser window size. There isn't agreement on which particular dimensions should be used, but a good number of web design pundits offer similar recommendations. For example:
-  http://960.gs/ (recommendation: width:960px of course)
- http://www.useit.com/alertbox/screen_resolution.html (recommendation: 1024x768, but test against 800x600, 1280×1024,1600×1200)
- http://www.smashingmagazine.com/2007/11/23/screen-resolutions-and-better-user-experience/ (comments show advocates for 1000px, 960px and 968px)

What I'm thinking:
- Allow authors to choose a particular canvas size both on the New File Dialog and offer an ability to change dimensions (probably via some UI on editor toolbar). We would offer a list of commonly used sizes (e.g., 1024x768) plus a custom width/height option
- The dark gray 5px border that we should now around the editing canvas would be replaced by some simple graphical frame that has an inner area equal to the current canvas size. (From an implementation perspective, maybe we can auto-construct a simple silhouette SVG that has the correct dimensions)
- Preview-in-browser would _not_ show the frame and just run directly in the browser

**Comment by peller** [comment]
I understand that it's important to test and perhaps even develop in a certain size, but isn't it just as important to use fluid layout that would work in any size?

**Comment by JonFerraiolo** [comment]
Yes, fluid layout is important, but accordion to various web articles, the recommended approach is optimize for a particular size and then test against other sizes to make sure you have proper fluid layout. That gets you gold stars. In practice, however, lots of web sites just optimize for a particular size and tough luck to users who have smaller screens either out of ignorance, laziness, lack of technical depth, or expediency. Note that Flash-based web sites do not reflow. Good reflow doesn't appear to be a top-level requirement for lots of sites.

**Comment by JonFerraiolo** [comment]
Pushing to M4.

**Comment by JonFerraiolo** [comment]
Pushing to M5

**Comment by JonFerraiolo** [comment]
Pushing to M6

**Comment by JonFerraiolo** [comment]
-> M7

**Comment by JonFerraiolo** [comment]
-> M8
